### PR TITLE
feat: header logo

### PR DESCRIPTION
**Pull request by Meolocious**
## Short description
This PR will introduce the logo image (not the small avatar) in the Header component. Also the style is updated with the latest requirements.

## List of changes proposed in this pull request
- removed "box" in the DialogBubble.tsx
- changed bg colour in the DialogBubble.tsx
- added "logo" as optional in the Title.tsx

## Screenshot

Before changes:
Send Header Before

After changes:
Send Header After

## How to test
Tested on the Storybook. Then builded and tested on B2BP. Due to the logo should be loaded from B2BP, here it could be visible only changing locally this file here:

src\components\Header\components\Title.tsx

```
import {
  Stack,
  Avatar,
  Typography,
  Chip,
  type AvatarProps,
  useTheme,
  Link,
  useMediaQuery,
} from '@mui/material';
import { type CommonProps } from '../../../types/components';
import logoImage from '../../../assets/images/logoSend.svg';

export interface TitleProps extends CommonProps {
  product: {
    name: string;
    href?: string;
  };
  avatar?: AvatarProps;
  beta?: boolean;
}

export const Content = ({
  avatar,
  beta,
  product: { name: productName, href: productHref },
  theme,
}: TitleProps) => {
  const { palette, spacing } = useTheme();
  const textColor =
    theme === 'dark' ? palette.primary.contrastText : palette.text.primary;
  const label = 'beta';

  const isBelow900px = useMediaQuery('(max-width:900px)');

  const productContent = logoImage ? (
    <img src={logoImage} alt="Logo" style={{ height: '70px' }} />
  ) : (
    <Typography
      color={textColor}
      fontSize={{ xs: spacing(3), sm: spacing(3.5) }}
      noWrap
      variant="h5"
    >
      <b>{productName}</b>
    </Typography>
  );

  return (
    <Stack direction="row" gap={1} alignItems="center">
      {avatar && <Avatar {...avatar} />}
      {productHref && (
        <Link
          href={productHref}
          underline="none"
          sx={{
            margin: isBelow900px ? '22px 24px 16px 0' : '22px 24px 16px 24px',
          }}
        >
          {productContent}
        </Link>
      )}
      {beta && (
        <Chip
          label={label}
          color="primary"
          sx={{ height: 20, width: 45 }}
          size="small"
        />
      )}
    </Stack>
  );
};
```

and putting this .svg inside the src\assets\images:

logoSend

### CODE REVIEW

File: src/components/DialogBubble.tsx [hunk 1]
```diff
@@ -31,22 +24,13 @@ const useStyles = (mui: Theme) => ({
   bubbleContainer: {
     transform: 'rotate(180deg)',
     position: 'absolute',
-    backgroundColor: mui.palette.primary.main,
-    marginTop: '30px',
+    backgroundColor: '#ffffff',
```
Comment (datalek) [inline src/components/DialogBubble.tsx:27]: Replace hardcoded value with theme value pls 🙏

**Review by datalek — CHANGES_REQUESTED** [review]

File: src/components/DialogBubble.tsx [hunk 2]
```diff
@@ -17,9 +11,14 @@ export const DialogBubble = ({
 }: DialogBubbleProps) => {
   const mui = useTheme();
   const styles = useStyles(mui);
+  const theme = useTheme();
```
Comment (kin0992) [inline src/components/DialogBubble.tsx:14]: what is the difference between line 14 and line 12?

**Comment by kin0992** [comment]
@Meolocious would you add a screenshot of the outcome?

File: src/components/DialogBubble.tsx [hunk 3]
```diff
@@ -17,9 +11,12 @@ export const DialogBubble = ({
 }: DialogBubbleProps) => {
   const mui = useTheme();
   const styles = useStyles(mui);
+  const white = mui.palette.common.white;
   return (
-    <Stack sx={styles.bubbleContainer} aria-haspopup="true">
-      <Box sx={styles.bubblePointer} />
+    <Stack
+      sx={{ ...styles.bubbleContainer, bgcolor: white }}
```
Comment (kin0992) [inline src/components/DialogBubble.tsx:17]:
```suggestion
      sx={{ ...styles.bubbleContainer, bgcolor: mui.palette.common.white }}
```
you are using it only here

File: src/components/DialogBubble.tsx [hunk 4]
```diff
@@ -17,9 +11,12 @@ export const DialogBubble = ({
 }: DialogBubbleProps) => {
   const mui = useTheme();
   const styles = useStyles(mui);
+  const white = mui.palette.common.white;
```
Comment (kin0992) [inline src/components/DialogBubble.tsx:14]:
```suggestion
```

**Review by kin0992 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**